ncj/3dsmax/vray-dr/job.template.json

```diff
@@ -199,7 +210,7 @@
                         },
                         "multiInstanceSettings":{
                             "numberOfInstances":"[parameters('nodeCount')]",
-                            "coordinationCommandLine":"cmd.exe /c if \"[parameters('vrayRenderer')]\" == \"VRayAdv\" (cmd.exe /c start \"\" \"C:\\Autodesk\\3dsMax2018\\vrayspawner2018.exe\" \"-port=[parameters('vrayPort')]\") else (cmd.exe /c start C:\\server\\3dsmaxVray\\bin\\vray.exe -server -portNumber=[parameters('vrayPort')])",
+                            "coordinationCommandLine":"cmd.exe /c if (\"[parameters('vrayRenderer')]\" == \"VRayAdv\") if (\"[parameters('maxVersion')]\" == \"3ds Max 2018\") cmd.exe /c start \"\" \"C:\\Autodesk\\3dsMax2018\\vrayspawner2018.exe\" \"-port=[parameters('vrayPort')]\" if (\"[parameters('maxVersion')]\" == \"3ds Max 2019\")   cmd.exe /c start \"\" \"C:\\Autodesk\\3dsMax2019\\vrayspawner2019.exe\" \"-port=[parameters('vrayPort')]\" if (\"[parameters('vrayRenderer')]\" == \"VRayRT\") if (\"[parameters('maxVersion')]\" == \"3ds Max 2018\")  cmd.exe /c start C:\\server\\3dsMax2018Vray\\bin\\vray.exe -server -portNumber=[parameters('vrayPort')])\"  if (\"[parameters('maxVersion')]\" == \"3ds Max 2019\") cmd.exe /c start C:\\server\\3dsMax2019Vray\\bin\\vray.exe -server -portNumber=[parameters('vrayPort')])\"",
                             "commonResourceFiles":[
```

Comment:
coordinationCommandLine in the DR template is impossible to make sense of. Can we pull this out into another script to make it easier to read?

Reply:
I think it's possible to pull them out.